integrations/walkme/lib/index.js

```diff
@@ -19,6 +19,7 @@ var WalkMe = module.exports = integration('WalkMe')
   .option('trackWalkMeEvents', false)
   .option('loadWalkMeInIframe', false)
   .option('integrityHash', '')
+  .option('customDirecotry', 'users')

```

Comment:
Is this a typo? Should it be `customDirectory`?

Reply:
thank you! yeah, this is totally a typo that autocomplete carried over the definition. Thanks for noticing!